src/components/multi-select/multi-select.md

```diff
@@ -0,0 +1,163 @@
+`MultiSelect` allows for the selection of multiple values in an accessible, searchable interface. It also provides flexibility in how you render selected options and options in the dropdown, but the default is MDS compatible and assured.
```

Comment:
```suggestion
See [design specifications.](https://www.notion.so/MultiSelect-9da6ee3245d946699106231cf161f783)

`MultiSelect` allows for the selection of multiple values in an accessible, searchable interface. It also provides flexibility in how you render selected options and options in the dropdown, but the default is MDS compatible and assured.
```

Reply:
Welcome to edit the document as well if you want to call out anything specific. Otherwise, a link is fine for now.